src/SDL12_compat.c

```diff
@@ -5581,6 +5581,24 @@ UpdateRect12to20(SDL12_Surface *surface12, const SDL12_Rect *rect12, SDL_Rect *r
     }
 }
 
+/* For manual throttling of screen updates. */
+static int
+GetDesiredMillisecondsPerFrame()
+{
+    SDL_DisplayMode mode;
+    if (VideoSurface12->flags & SDL12_FULLSCREEN) {
+       if (!SDL20_GetWindowDisplayMode(VideoWindow20, &mode)) {
+           return 1000 / mode.refresh_rate;
```

Comment:
Need to check mode.refresh_rate is not 0 here like you do below.

Reply:
Whoops! Fixed.